Modules/Quotas/src/Quotas.Infrastructure/Persistence/IServiceCollectionExtensions.cs

```diff
@@ -45,9 +47,14 @@ public static void AddDatabase(this IServiceCollection services, Action<DbOption
                 throw new Exception($"Unsupported database provider: {options.Provider}");
 
         }
-        services.AddTransient<ITiersRepository, TiersRepository>();
         services.AddTransient<IIdentitiesRepository, IdentitiesRepository>();
+        services.AddTransient<IMessagesRepository, MessagesRepository>();
         services.AddTransient<IMetricsRepository, MetricsRepository>();
+        services.AddTransient<IMetricStatusesRepository, MetricStatusesRepository>();
+        services.AddTransient<ITiersRepository, TiersRepository>();
+
+        services.AddTransient<IMetricCalculatorFactory, MetricCalculatorFactory>();
+        services.AddTransient<NumberOfSentMessagesMetricCalculator>();
```

Comment:
Why do you explicitly add this?

Reply:
Autofac requires all of those

Reply:
The `NumberOfSentMessagesMetricCalculator`? It shouldn't actually, as far as I know. Are you 100% sure?